thirdeye/thirdeye-pinot/src/main/java/com/linkedin/thirdeye/dashboard/resources/AnomalyFunctionResource.java

```diff
@@ -19,6 +19,7 @@
 import org.slf4j.Logger;
 import org.slf4j.LoggerFactory;
 
+@Path("thirdeye")
```

Comment:
Is "thirdeye" the apt path prefix here? Could we just remove "metadata" from the 2 calls below and put it here?